src/Tizen.Security.SecureRepository/Tizen.Security.SecureRepository/SafeAliasListHandle.cs

```diff
@@ -21,12 +21,12 @@
 
 namespace Tizen.Security.SecureRepository
 {
-    internal class SafeAliasListHandle
+    internal class SafeAliasContainerHandle<T> where T : ICollection<string>, new() 
```

Comment:
Why do you make as generic type? it just use for once, it seem to not need a generic. 
You just change HashSet from List, it is enough

Reply:
Currently it is used in two places, as the base class for the SafeAliasListHandle which is still used as outlined in the pr description, and in DataManager where I already changed the behaviour. I did it this way because I wanted the other usages to still work if they return the same values multiple times. I assume KeyManager and CertificateManager also return unique values? Then this is indeed not needed and I can adapt it.